test/calibration/experiments/test_drag.py

```diff
@@ -56,32 +90,41 @@ def test_reps(self):
     def test_end_to_end(self):
         """Test the drag experiment end to end."""
 
-        backend = DragBackend(gate_name="Drag(xp)")
+        calc_parameters = {"gate_name": "Drag(xp)", "ideal_beta": 2.0, "error": 0.03}
+        backend = MockIQBackend(
+            compute_probabilities=compute_probability, calculation_parameters=[calc_parameters]
```

Comment:
I thought, perhaps, you could use `kwargs` in `__init__`, as long as you set the type hints correctly

Reply:
Yea. I think it is better to add `kwargs` after this PR will be merged as it become very big

Reply:
Good Idea, can I open an issue and I will fix it after the merge?

Reply:
changed design so this is not relevant anymore